transform.go

```diff
@@ -66,11 +66,8 @@ func newTransform(node *p2p.QriNode, ds *dataset.Dataset, infile cafs.File) *tra
 // ExecFile executes a transformation against a starlark file located at filepath, giving back an EntryReader of resulting data
```

Comment:
Update this comment. No longer uses "filepath", doesn't give back an EntryReader.